src/utils/data/setup-store.ts

```diff
@@ -7,7 +7,13 @@ import { schema as schemaDefinition } from './schema';
 
 declare const __ENV_API_HOST__: string;
 
-export function setupStore(appState: AppState): { store: Store; local?: IndexedDBStore; coordinator?: Coordinator } {
+interface StoreSetup {
+  store: Store;
+  local?: IndexedDBStore;
+  coordinator?: Coordinator;
+}
+
+export function setupStore(appState: AppState): StoreSetup {
```

Comment:
Is this change necessary for anything?

Reply:
Hmm no, I remember I did this change because I tried to export as well the API store. That didn't work out so I'll remove this change as it's not necessary.